helm/templates/envoy-defines.yaml

```diff
@@ -0,0 +1,37 @@
+{{- define "nginx-for-s3.envoy-container" }}
+        - name: envoy
+          image: envoyproxy/envoy:v1.18.3
+          volumeMounts:
+          - readOnly: true
+            mountPath: /config
+            name: proxy-config
+          args:
+          - "envoy"
+          - "--config-path"
+          - "/config/envoy.yaml"
+          ports:
+            - name: http
+              containerPort: {{ .Values.ports.envoy.port }}
+              protocol: TCP
+            - name: admin
+              containerPort: 8001
+              protocol: TCP
+          # livenessProbe:
```

Comment:
any use for this comments?

Reply:
These health-checks cause the B2B to fail but it should be solved because we want the health-checks.
Hopefully raster will solve this problem.